core/src/main/java/com/netflix/conductor/core/execution/ApplicationException.java

```diff
@@ -47,24 +47,39 @@ public int getStatusCode(){
 	}
 	
 	private Code code;
-	
+
+	public boolean getRetryable() {
+	    return retryable;
+	}
+
+	private boolean retryable;
+
+	private void checkIsRetryable() {
```

Comment:
Not need to introduce retryable field, change this to isRetryable and make it public:

```
public boolean isRetryable() {
    return this.code == Code.BACKEND_ERROR;
}
```